credentials.go

```diff
@@ -69,31 +69,37 @@ func (endpoint *CloudEndpoint) ResolveEndpoint(ctx context.Context, params s3.En
 }
 
 func s3Client(ctx context.Context, cfg Config) (*s3.Client, error) {
+	var endpointResolver s3.EndpointResolverV2
 	var awsAccessKey string
 	var awsSecretKey string
-	var endpoint s3.EndpointResolverV2
+	region := "us-east-2"
 
 	if cfg.IsLocal {
 		awsAccessKey = "AKIAIOSFODNN7EXAMPLE"
 		awsSecretKey = "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
-		endpoint = &LocalEndpoint{Bucket: cfg.BucketName, S3Port: cfg.Local.BucketPort}
+		endpointResolver = &LocalEndpoint{Bucket: cfg.BucketName, S3Port: cfg.Local.BucketPort}
 	} else {
 		awsAccessKey = cfg.Cloud.AWS.AccessKey
 		awsSecretKey = cfg.Cloud.AWS.SecretKey
-		endpoint = &CloudEndpoint{Address: "s3-us-east-2.amazonaws.com", Bucket: cfg.BucketName}
```

Comment:
I'm a bit confused, but where do you set the bucket?

Reply:
The bucket is a required parameter in the get / put / list requests:

params := &s3.PutObjectInput{
		Bucket: aws.String(cfg.BucketName),
		Key:    aws.String(key),
		Body:   file,
	}
	
The s3Client has a default resolver, we only need to set the resolver to something custom when we have our own endpoint

Reply:
So do we assume that all cloud endpoints are always aws s3?

Reply:
Good point, I'll put back the endpoint line. Using the default resolver was testing if it was incorrect due to the 301 upload error